app.rb

```diff
@@ -7,31 +7,37 @@
 require "pg"
 
 class Memo
-  attr_reader :title, :about
-  def initialize(title, about)
-    @title, @about = title, about
+  attr_reader :id, :title, :about
+  def initialize(id, title, about)
+    @id = id
+    @title = title
+    @about = about
+  end
+
+  def self.connect_db
+    @@conn = PG.connect(user: "postgres", dbname: "memo_app")
```

Comment:
これだとメソッドが呼ばれるたびに接続してしまうので、下記の方がいいかもです。

```suggestion
    @@conn ||= PG.connect(user: "postgres", dbname: "memo_app")
```